.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Provider Issue
+    url: https://github.com/terraform-providers
+    about: Each provider has its own repository, and issues should be opened there not on the main Terraform repository.
+  - name: Question
+    url: https://www.terraform.io/community.html
```

Comment:
We've now standardized on linking to [our official community forum](https://discuss.hashicorp.com/c/terraform-core) as the primary place to ask questions, in the interests of giving a more clearer direction.

What do you think about linking there instead, and changing the text to say "If you have questions about Terraform usage, please start a topic in the community forum."? That uses a similar phrasing to some of our hashibot close-issue messages.

Reply:
I don't have strong feelings about it. The only reason I didn't do that was to avoid having to add 2 more links to each relevant category (Core, SDK, Providers) as we don't really have generic "Terraform category" and we don't want to point people to the wrong one.

If you are ok with that though I can just replace this with:

```yaml
 - name: Core/Usage Question
    url: https://discuss.hashicorp.com/c/terraform-core
    about: If you have questions about Terraform usage, please start a topic in the community forum.
 - name: Provider Question
    url: https://discuss.hashicorp.com/c/terraform-providers
    about: If you have questions about anything related to specific Terraform provider (such as AWS, Google, AzureRM etc.), please start a topic in the community forum.
 - name: Plugin SDK Question
    url: https://discuss.hashicorp.com/c/terraform-providers/tf-plugin-sdk
    about: If you have questions about Terraform Plugin development and the SDK, please start a topic in the community forum.
```
and I can put something very similar to the SDK repo.

Reply:
That's an interesting wrinkle... thanks for pointing that out. I would lean slightly towards keeping things simpler and just linking to the "Terraform Core" category and then using moderation in the forum to move things around as necessary, since I think people are more likely to use the forum rather than GitHub if we give them one clear option, but that could potentially lead to more forum moderation work.

Perhaps we could compromise by generalizing the existing "Provider Issue" link to _also_ encompass questions -- some phrasing that means "anything that is specific to a provider" -- and then let the provider repositories themselves have a similar link out to the appropriate community forum category or whatever other forums make sense there. Hopefully that could avoid us needing to flatten out all of the decisions into the top-level here.

So I guess that would give us the following items on the page:
* Terraform Core Bug Report (a template)
* Terraform Core Feature Request (a template)
* Terraform Language or Workflow Questions (link to https://discuss.hashicorp.com/c/terraform-core)
* Provider-related Feedback and Questions (link to https://github.com/terraform-providers)
* Provider Development Feedback and Questions (link to https://github.com/hashicorp/terraform-plugin-sdk/issues/new/choose with a similar set of SDK-specific templates and links)

It does mean a few more hops for folks whose concerns are about provider usage or provider development, but on the plus side it also reinforces up front that Terraform Core is a separate project from these other parts and so hopefully along the way helps users develop some intuition about how these components are split up.